### ISSUE TITLE
Remove the white bar below the keys.

### ISSUE THREAD

**Issue by ciaociaoweb** [SimpleMobileTools/Simple-Keyboard#156]
![Screenshot_20230404-071052_GitHub.jpg](https://user-images.githubusercontent.com/105258592/229664499-6663315d-77cd-40ca-8dfa-4bdb58dfb942.jpg)

**Comment by tibbi** [comment]
that is handled by your system, not us. Does it work differently in other keyboards?

**Comment by ciaociaoweb** [comment]
Yes. But with other keyboards the bar matches the theme of the keyboard. Here it stays white no matter the colour of the keyboard. Other keyboards match the colour of the bar. Is that something that you could add?

**Comment by tibbi** [comment]
what color you have set in the app Settings -> Customize colors -> Navigation bar color?

**Comment by Merkost** [comment]
Probably the phone theme is light, isn't it? @fard33n

**Comment by fard33n** [comment]
No, it's dark.

**Comment by stateandrei5** [comment]
Mine is black and i hate it
